aspnetcore/blazor/hosting-models.md

```diff
@@ -241,13 +303,33 @@ To avoid server-based APIs for Blazor WebAssembly apps, adopt Blazor Server, whi
 
 ### Small payload size with fast initial load time
 
+:::moniker range=">= aspnetcore-8.0"
+
+Components rendered for the Blazor Server hosting model have relatively small payload sizes with faster initial load times. When a fast initial load time is desired, adopt rendering for the Blazor Server hosting model.
+
+:::moniker-end
+
+:::moniker range="< aspnetcore-8.0"
+
 Blazor Server apps have relatively small payload sizes with faster initial load times. When a fast initial load time is desired, adopt Blazor Server.
 
+:::moniker-end
+
 ### Near native execution speed
 
-Blazor Server apps generally execute on the server quickly. However, Blazor Server apps are usually slower than other types of apps that execute natively on the client.
+:::moniker range=">= aspnetcore-8.0"
 
-:::moniker range=">= aspnetcore-6.0"
+Components rendered for the Blazor Server hosting model generally execute on the server quickly. However, the components are usually slower than other types of apps that execute code natively on the client.
```

Comment:
Hmm, this statement about server vs client execution seems incorrect. Often components run faster on the server because the server hardware is more powerful than the client hardware.

In general .NET runtime execution is highly optimized for both server and client platforms. It's on the web platform (browsers) that we have challenges. .NET runtime execution in browsers is much faster in .NET 8 thanks to the new partial JIT support in the interpreter and AOT helps further, but it's still slower than when running .NET on other platforms and generally slower than the equivalent JavaScript code.

Reply:
I didn't write this, and I don't recall if the original intent had anything to do with server load or network latency, which would make this statement make more sense I think. Whatever the case, I'm not sure if these remarks are of much value here. Should they be stricken 🔪?

Reply:
I think we want to revise this "Near native execution speed" section to basically say that Blazor Server and Blazor Hybrid are fast. Blazor WebAssembly is not as fast but can be made faster using AOT compilation.

Reply:
I'll work on this further in the morning. Thus far, I just removed the first paragraph. It seems like the two paragraphs that remain are close to your suggestion.